libs/ui/layout/src/lib/value-list/value-list.component.html

```diff
@@ -0,0 +1,44 @@
+<div
```

Comment:
maybe this component should benefit from a reuse of what has already been done for the long text cell PR.